.github/workflows/integration.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
           kubectl exec $(kubectl get pods -l app=mysql -o jsonpath='{.items[0].metadata.name}') -- mysql -uroot -h 127.0.0.1 -ppass -e "create table myapp.ignore_me (c int);"
           
           # Apply the desired schema and wait for it to be ready.
-          kubectl apply -k config/integration
+          kubectl apply -f config/integration/schema
```

Comment:
what's difference between `k` and `f` ?
<img width="632" alt="Screenshot 2023-05-28 at 10 44 13" src="https://github.com/ariga/atlas-operator/assets/12751435/72244300-02e3-4082-b7cf-1a19b0919939">

Reply:
`-k` is used to apply 'kustomize.yaml'. Kustomize is a tool similar to Helm charts, but we are currently not using any advanced features of it